### PR TITLE
build/configs: Update Wi-Fi driver config for esp board

### PR DESCRIPTION
**Pull request by zhouxinhe**
- Select `SELECT_PROPIETARY_WLAN` instead of `SELECT_NO_DRIVER`
- Fix typo in virtual_wifi_utils.c

### CODE REVIEW

**Comment by seinfra** [comment]
Target : [991c09c979fa9b913c973cae0eba81ee78d16d99] - Code Rule Check OK.

**Comment by seinfra** [comment]
Target : [991c09c979fa9b913c973cae0eba81ee78d16d99] - Code Rule Check (C++) OK.

File: build/configs/esp_wrover_kit/security/defconfig [hunk 1]
```diff
@@ -739,10 +739,10 @@ CONFIG_TLS_MPI_MAX_SIZE=512
 CONFIG_WIFI_MANAGER=y
 # CONFIG_SLSI_WPA_SUPPLICANT is not set
 CONFIG_SELECT_PROPIETARY_SUPPLICANT=y
-CONFIG_SELECT_NO_DRIVER=y
+# CONFIG_SELECT_NO_DRIVER is not set
 # CONFIG_SELECT_SCSC_WLAN is not set
 # CONFIG_SELECT_RTK_WLAN is not set
-# CONFIG_SELECT_PROPIETARY_WLAN is not set
+CONFIG_SELECT_PROPIETARY_WLAN=y
 CONFIG_WIFIMGR_SOFTAP_IFNAME="wl1"
```
Comment (ChinaTizenRT) [inline build/configs/esp_wrover_kit/security/defconfig]: it looks good for me,  latest wifi manager module add this new config item.

**Comment by zhouxinhe** [comment]
@pillip8282 @SeonghoByeon @xixidodo 
Could you please help to review this PR?

**Pull request merged**